docs/proposals/get-live-resources-from-cli.md

```diff
@@ -0,0 +1,168 @@
+---
+title: Get Live Resources from CLI
+authors:
+  - "@cjcocokrisp" 
+sponsors:
+  - TBD        
+reviewers:
+  - TBD
+approvers:
+  - TBD
+
+creation-date: 2025-06-17
+last-updated: 2025-06-17
+---
+
+# Get Live Resources from the ArgoCD CLI
+
+Add the ability to get the live manifest of a resource from the ArgoCD CLI.
+
+## Open Questions [optional]
+
+- Command sepcific naming convention. (See Implementation Details for what I'm thinking)
```

Comment:
I think we can follow the naming conventions used by some sibling commands like `argocd app get | resources`. Use long flag names (like `--resource-name, --app-namespace`)  and don't support short flag (like `-x`) except for most common ones.

```bash
$ argocd app get -h

Flags:
  -N, --app-namespace string   Only get application from namespace
      --hard-refresh           Refresh application data as well as target manifests cache
  -h, --help                   help for get
  -o, --output string          Output format. One of: json|yaml|wide|tree (default "wide")
```